src/extensions/cellpack/state.ts

```diff
@@ -148,3 +294,27 @@ const StructureFromAssemblies = PluginStateTransform.BuiltIn({
         b?.data.customPropertyDescriptors.dispose();
     }
 });
+
+const CreateTransformer = StateTransformer.builderFactory('cellPACK');
+export const CreateSphere = CreateTransformer({
```

Comment:
Please adjust all the names and labels to include 'membrane' so it is clear what representation this creates, thanks.